phobos/smurf/sensors.py

```diff
@@ -172,7 +172,9 @@ def __init__(self, robot=None, name=None, targets=None, **kwargs):
 
         targets = [robot.get_joint(t) if type(t) is str else t for t in targets]
 
-        assert all([isinstance(t, representation.Joint) for t in targets])
+        if not all([isinstance(t, representation.Joint) for t in targets]):
+            print(targets)
+            raise AssertionError("Parsed invalid joint")
```

Comment:
Dieser Check sollte bei den anderen Sensoren ggf auch noch ergänzt werden.

Reply:
Wurden ergänzt, habe dei anderen "asserts" in "if not" ebenfalls umgeschrieben. Erinere mich nicht mehr daran was das Problem mit "assert" war. Oder rest so lassen und nur die ohne check updaten ?

Reply:
Es gibt kein Problem mit assert, außer das wir so mehr debugging infos kriegen